draft-ietf-cats-usecases-requirements.md

```diff
@@ -770,6 +770,134 @@ informative:
    WAN1 -> Cloud2 vCPE1 -> Cloud2 APP1
 
 
+##  Computing-Aware AI large model
+   
+   AI large model refers to a type of artificial intelligence model that 
+   is trained on massive amounts of data using deep learning techniques. 
+   These models are characterized by their large size, high complexity, 
+   and high computational requirements. AI large models have become 
+   increasingly important in various fields, such as natural language 
+   processing, computer vision, and speech recognition.
+   
+   There are usually two types of AI large models, AI foundation model 
+   and customized model. AI foundation large model is a model that can 
+   handle multiple tasks and domains, and has wider applicability and 
+   flexibility, but may not perform as well as customized model in 
+   specific domain tasks. Customized model is trained for specific 
+   industries or domains, and more focused on solving specific problems, 
+   but may not be applicable to other domains. AI foundation model 
+   usually involve mega-scale parametere, while customized model involves 
+   large or middle-scale parameters.
+   
+   Also, AI large model contains two key phases: training and inference. 
```

Comment:
```suggestion
   Also, AI large model involves two key phases: training and inference. 
```

Is this really specific to these models?

Reply:
Good question. Actually, training is a universal phase that each AI model need to do, while inference is not. AI large model, in most cases refers to large language model, involves inference. But for some other AI models, like a model trained to categorize images, inference is not needed.

Reply:
I am not sure about that. From [Google AI model inference](https://www.google.com/search?q=ai+model+inference):
`Machine learning inference is the process of running live data into a machine learning algorithm to calculate output such as a single numerical score.`

It seems categorize images is also inference.

Maybe the difference between the large model and traditional ai model is that it can generate a lot of content(so called AIGC: AI Generated Content). But I am not sure about what the difference it makes related to CATS.

Reply:
By looking at different citations, I think this one from [xilinx](https://www.xilinx.com/applications/ai-inference/difference-between-deep-learning-training-and-inference.html#:~:text=Machine%20learning%20inference%20is%20the,sort%20different%20fruits%20by%20color.) is worth consideration

Machine learning inference is the ability of a system to make predictions from novel data.

So, in this case, I agree classifying images is an inference, since a machine learning model trained to classify images of cats and dogs can be used to infer whether a new image is of a cat or a dog.

And so, inference can be a common ability of AI models.

> Maybe the difference between the large model and traditional ai model is that it can generate a lot of content(so called AIGC: AI Generated Content).

In my view, why AI large model has billions of parameters is because it deals with natural languages.

> But I am not sure about what the difference it makes related to CATS.

I think if an AI large model aims to generate images or videos, then it will often bring on high demand on network resource and computing resource, which is the area related to CATS.